ethereum/core/src/jmh/java/org/hyperledger/besu/ethereum/vm/operations/CountLeadingZerosOperationBenchmark.java

```diff
@@ -78,11 +89,23 @@ public void setUp() {
   }
 
   @Benchmark
+  @OperationsPerInvocation(OPERATIONS_PER_INVOCATION)
+  @Warmup(iterations = 3, time = 500, timeUnit = TimeUnit.MILLISECONDS)
   public void executeOperation() {
-    for (int i = 0; i < 1000000; i++) {
+    for (int i = 0; i < OPERATIONS_PER_INVOCATION; i++) {
       frame.pushStackItem(bytes);
       CountLeadingZerosOperation.staticOperation(frame);
       frame.popStackItem();
     }
   }
+
+  @Benchmark
+  @OperationsPerInvocation(OPERATIONS_PER_INVOCATION)
+  @Warmup(iterations = 1, time = 500, timeUnit = TimeUnit.MILLISECONDS)
```

Comment:
Seems odd to me to warmup the baseline differently to the test?

Overall, to keep it simple to reason about, I'd rather we choose sufficient warmup settings to cover all cases. 
If it's a major issue between different opcodes then could maybe a slow and a fast category?

I feel like tailoring every opcode precisely to save a few seconds is going to make this exercise onerous for most devs. Tailoring the baseline differently to the test feels too precise.

Reply:
If the overall suite ends up taking many minutes/hours anyway, I don't think there's much to be gained from this level of precision.